penscratch-2/css/editor-blocks.css

```diff
@@ -19,560 +19,6 @@ Description: Used to style Gutenberg Blocks in the editor.
 1.0 General Typography
 --------------------------------------------------------------*/
 
-.edit-post-visual-editor .editor-block-list__block,
-.editor-default-block-appender textarea.editor-default-block-appender__content {
-	font-family: "Roboto Slab", Georgia, Times, serif;
+body {
+	background: #fff;
```

Comment:
It's kind of a shame that this line is needed just to override the default background color, otherwise we could get rid of this file entirely.

Reply:
We could add a Gutenberg class to the CSS that targets the site content, but I don't want to start mixing editor CSS into the main file.